config.toml.example

```diff
@@ -329,6 +329,12 @@
 # Defaults to rust.debug-assertions value
 #debug-assertions-std = false
 
+# Whether or not to leave debug! and trace! calls in the rust binary.
+# Overrides the `debug-assertions` option, if defined.
+# 
+# Defaults to rust.debug-assertions value
+#debug-logging = true
```

Comment:
This should match the default:
```suggestion
#debug-logging = debug-assertions
```

Reply:
I was copying how other things with defaults work, like debug-assertions-std above

Reply:
Ok, can you change the comments for those too? It's confusing for the comment to sometimes be the default but other times be a recommended setting; recommendations should go in the comments above.

Reply:
@jyn514 yeah I can change those!

I just realized something
if debug-assertions is true, and debug_logging is false, then logs will still happen... 
We need to set max_level_info (not just release_max_level_info) (https://github.com/tokio-rs/tracing/blob/master/tracing/src/level_filters.rs#L74) in that case right?

Reply:
Yes, that sounds right. Good catch!